src/index.js

```diff
@@ -179,22 +179,55 @@ class KadDHT {
    *
    * @param {Buffer} key
    * @param {Buffer} value
+   * @param {Object} options - get options
+   * @param {number} options.minPeers - minimum peers that must be put to to consider this a successful operation
+   * (default: 0)
```

Comment:
We cannot have this default with this logic. When this was suggested https://github.com/libp2p/js-libp2p-kad-dht/issues/68#issuecomment-452390478, it was to keep the current behaviour as default. That is, the default should be the number of closestPeers.

I believe that what jacob was suggesting was that we should keep the default of 0 fails to put.

Reply:
Right now, the behavior succeeds if you find NO peers to put to.  It will fail if you request a put to any peer, and it fails.

The way the logic is written, if you put 0 (or put nothing), it will not fail if you fail to put to any peers.  It will continue to fail as it did before if any individual peer put fails.  

If you put 1, the logic now requires at least 1 successful put.  No peers will fail, as will finding 3 peers, each of which fails.

Is this what you were thinking it did?  Or is the variable just poorly named?

Reply:
@jacobheun did you by chance look at this pr?  do you remember your comment about minPeers?   happy to modify with your or @vasco-santos advice.  just let me know!

Reply:
Ok, I previously misunderstood the logic, we'll need to update this PR.

**Current functionality**
1. Get the closest peers
1. For each peer, put to that peer.
1. If **any** put fails, the request fails.

So the current logic is that `minPeers === closestPeers.length`. So once we get the closest peers, if `minPeers` is not set, it should be set to the number of peers returned.